info/Team/contributors.json

```diff
@@ -7,6 +7,70 @@
       "avatar": "https://avatars.githubusercontent.com/u/10688662?v=4",
       "githubLink": "https://github.com/arjundevarajan",
       "date": "2023/07/18"
+    },
+    {
+      "realName": "Zhou Lixian",
+      "githubId": "zhoulixiandevlake",
+      "email": "",
+      "avatar": "https://avatars.githubusercontent.com/u/136300268?v=4",
+      "githubLink": "https://github.com/zhoulixiandevlake",
+      "date": "2023/07/03"
+    },
+    {
+      "realName": "Eliad Cohen",
+      "githubId": "EliadCohen",
+      "email": "",
+      "avatar": "https://avatars.githubusercontent.com/u/11726079?v=4",
+      "githubLink": "https://github.com/EliadCohen",
+      "date": "2023/07/03"
+    },
+    {
+      "realName": "Albin Vass",
+      "githubId": "albinvass",
+      "email": "",
+      "avatar": "https://avatars.githubusercontent.com/u/16024592?v=4",
+      "githubLink": "https://github.com/albinvass",
+      "date": "2023/07/04"
+    },
+    {
+      "realName": "Jacob Cook",
+      "githubId": "peakwinter",
+      "email": "",
+      "avatar": "https://avatars.githubusercontent.com/u/997174?v=4",
+      "githubLink": "https://github.com/peakwinter",
+      "date": "2023/07/27"
+    },
+    {
+      "realName": "Ethan Davidson",
+      "githubId": "Ethan Davidson",
```

Comment:
Is this the real name, not id?

Reply:
Thanks, I fixed in the latest commit below.